main.js

```diff
@@ -0,0 +1,191 @@
+// selet elements in DOM
+
+const form = document.querySelector("#itemForm");
+const itemInput = document.querySelector("#itemInput");
+const itemList = document.querySelector("#itemList");
+const messageDiv = document.querySelector("#message");
+const clearButton = document.querySelector("#clearBtn");
+const filters = document.querySelectorAll(".nav-item");
+
+// create an empty array for tasks(items).
+let todoItems = [];
+
+// showing alert when we add item, or errors.
+const showAlert = function (message, msgClass) {
+  //msgClass = type of message, ie. success, danger, error, etc.
+  console.log("msg");
+  messageDiv.innerHTML = message;
+  messageDiv.classList.add(msgClass, "show");
+  messageDiv.classList.remove("hide");
+  setTimeout(() => {
+    messageDiv.classList.remove("show", msgClass);
+    messageDiv.classList.add("hide");
+  }, 2000);
+  return;
+};
+
+// filter tab tasks
+const getFilteredItems = function (type) {
```

Comment:
when we use get we expect a return value, so naming a void returning function with get prefix doesn't make sense. It should have been filterItem().